src/main/java/seedu/address/model/person/NameContainsKeywordsDoctorPredicate.java

```diff
@@ -38,7 +39,10 @@ public boolean equals(Object other) {
                 (NameContainsKeywordsDoctorPredicate) other;
         return keywords.equals(otherNameContainsKeywordsPredicate.keywords);
     }
-
+    @Override
+    public int hashCode() {
+        return Objects.hash(keywords);
+    }
     @Override
```

Comment:
Empty line before the override statement

Reply:
ok, just done